setup.py

```diff
@@ -13,6 +13,7 @@
     entry_points={
         'console_scripts': [
             'scenario-player=scenario_player.__main__:main',
+            'sp-log-packer=scenario_player.__main__:pack_logs',
```

Comment:
Since you already defined `pack-logs` as a subcommand on the `main` group this isn't really necessary.

Reply:
I...did? Good to know! :smile:

Reply:
`main()` is defined as a `@click.group()` that means any command that get's added via `@main.command()` will become a subcommand.